plugins/@grouparoo/spec-helper/src/lib/specHelper.ts

```diff
@@ -238,9 +238,17 @@ export namespace helper {
               key: "password",
               required: false,
             },
+            {
+              key: "failRemoteValidation",
```

Comment:
Because `failRemoteValidation` is something special only for the apps created in the spec-helper, it might be helpful to signal that this isn't a real/core thing by calling it `_ failRemoteValidation`. or something. 

I had thought this was a new concept you had introduced to `@grouparoo/core`